debezium-core/src/main/java/io/debezium/pipeline/signal/channels/process/SignalWriter.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.pipeline.signal.channels.process;
+
+import io.debezium.pipeline.signal.SignalRecord;
+
+/**
+ * Interface for writing signals to a channel.
+ */
+public interface SignalWriter {
```

Comment:
What do you think to have rename it to `SignalChannelWriter` to maintain consistency with the reader counterpart?

Reply:
Sure, done